src/tree/containerizedFunctionApp/ResolvedContainerizedFunctionAppResource.ts

```diff
@@ -139,18 +141,24 @@ export class ResolvedContainerizedFunctionAppResource extends ResolvedFunctionAp
     }
 
     public async deleteTreeItemImpl(context: IActionContext): Promise<void> {
+        const wizardContext: DeleteFunctionappWizardContext = Object.assign(context, {
+            site: this.site,
+            proxyTree: this as unknown as ContainerTreeItem,
+            ...(await createActivityContext())
+        });
+
         const message: string = localize('ConfirmDeleteFunction', 'Are you sure you want to delete function app "{0}"?', this.site.name);
-        const proxyTree: ContainerTreeItem = this as unknown as ContainerTreeItem;
-        const deleting: string = localize('DeletingFunctionApp', 'Deleting function app "{0}"...', this.site.name);
-        const deleteSucceeded: string = localize('DeleteFunctionAppSucceeded', 'Successfully deleted function app "{0}".', this.site.name);
-        await context.ui.showWarningMessage(message, { modal: true, stepName: 'confirmDelete' }, DialogResponses.deleteResponse);
-        await window.withProgress({ location: ProgressLocation.Notification, title: deleting }, async (): Promise<void> => {
-            ext.outputChannel.appendLog(deleting);
-            const client = await createWebSiteClient([context, proxyTree.subscription]);
-            await client.webApps.delete(nonNullValueAndProp(this.site, 'resourceGroup'), nonNullValueAndProp(this.site, 'name'));
-            void window.showInformationMessage(deleteSucceeded);
-            ext.outputChannel.appendLog(deleteSucceeded);
+        const title: string = localize('DeleteFunctionApp', 'Delete function app');
```

Comment:
I think it might look nice to add the app name to the activity title here here :)